### ISSUE TITLE
Reversed end diastolic flow in the fetal middle cerebral artery

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7372]
**Preferred term label:**
Reversed end diastolic flow in the fetal middle cerebral artery

**Synonyms**

**Definition (free text, please give PubMed ID)**
This appears to be rare but should we create a term?
https://www.ncbi.nlm.nih.gov/labs/pmc/articles/PMC4760589/

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
